src/scss/includes/panels/poi_panel.scss

```diff
@@ -13,8 +13,13 @@ $BLOCK_ICON_FONT_SIZE: 16px;
 
 .poi_panel__content {
   animation: appear 600ms forwards;
-  padding: 20px 30px 10px 30px;
+  padding: 20px 14px 0;
```

Comment:
we can use $spacing-xxs here too :)

Reply:
I think you mean `$spacing-l` to replace the `20px`.
I'll do it, but `14px` will remain as it isn't a standard spacing constant.